Examples/UIExplorer/js/TextInputExample.android.js

```diff
@@ -623,4 +710,22 @@ exports.examples = [
     title: 'Toggle Default Padding',
     render: function(): ReactElement { return <ToggleDefaultPaddingExample />; },
   },
+  {
+    title: 'Text selection & cursor placement',
+    render: function() {
+      return (
+        <View>
+          <SelectionExample
+            style={styles.default}
+            value="text selection can be changed"
+          />
+          <SelectionExample
+            multiline
```

Comment:
Not sure exectly what you mean here. The example is a copy paste of the iOS one (for some reason this example is probably the only one split in 2 files).

The examples is just a bunch of buttons to change the selection for a single line and multiline text input.